src/main/controllers/SubscriptionManagementController.ts

```diff
@@ -1,7 +1,34 @@
 import { Request, Response } from 'express';
+import { SubscriptionActions } from '../resources/actions/subscriptionActions';
 
 export default class SubscriptionManagementController {
   public get(req: Request, res: Response): void {
-    res.render('subscription-management');
+    const subscriptionActions = new SubscriptionActions();
+    const caseTableData = subscriptionActions.generateCaseTableRows();
+    const courtTableData = subscriptionActions.generateCourtTableRows();
+    let activeAllTab, activeCaseTab, activeCourtTab;
+    switch (Object.keys(req.query)[0]) {
```

Comment:
Only minor, however I think we can remove setting the 'false' values in each of the case statements, by setting the variables on line 9 initially to false.